test/standalone/Soil/soil_test_3d.jl

```diff
@@ -415,3 +415,49 @@ end
     ]) / ν < 1e-11
 
 end
+
+
+@testset "Lateral flow flag" begin
+    FT = Float32
```

Comment:
Should we test for Float32 and Float64?

Reply:
I think here it is OK because we are testing that the function does not change dY when there should be no horizontal components added to dY, and it isnt really related to carrying out computations and making sure the type isnt change by them?

Reply:
Good point